### ISSUE TITLE
HTML doesn't render properly on GH-Pages

### ISSUE THREAD

**Issue by lukeholman** [workflowr/workflowr#175]
Hi there,

I **love** workflowr, so that thanks for all your hard work on this excellent package.

I've never had this issue with workflowr before, so I'm not sure what to do (it's always just worked great for earlier projects). I am about to re-submit a paper, and I cannot get the Github pages site working nicely. It's here: https://lukeholman.github.io/W_shredder_model/, and the repo for it is here: https://github.com/lukeholman/W_shredder_model. The HTML looks great when I open it locally in my browser (presumably it will work fine for you too if you clone the repo and click 'docs/index.html'), but the Github pages version looks weird (not rendered, I guess). It's been on GHP for >12 hours, so it's not that. Any ideas?

Thanks!

**Comment by pcarbo** [comment]
@lukeholman The first thing I notice is that your `docs/site-libs` directory is missing. For some reason these files were not committed and/or pushed to the remote.

Also, I would recommend adding a URL to the website at the top of [your repo](https://github.com/lukeholman/W_shredder_model) so that people know how to get to the website if they end up there.

See the [wflow-divvy repo](https://github.com/lukeholman/W_shredder_model) for an example of what the final repository should look like.

Hope that helps.

**Comment by jdblischak** [comment]
> I love workflowr, so that thanks for all your hard work on this excellent package.

@lukeholman Thanks! We appreciate the feedback :-) I like the organization of your site; it's a great example of reproducible code accompanying a scientific paper!

> The HTML looks great when I open it locally in my browser (presumably it will work fine for you too if you clone the repo and click 'docs/index.html'), but the Github pages version looks weird (not rendered, I guess). It's been on GHP for >12 hours, so it's not that. Any ideas?

I agree with @pcarbo that the main issue is that the supporting CSS/JS files are not in the repository:

> The first thing I notice is that your docs/site-libs directory is missing.

But I'm not sure how that could have happened. I forked your repo and published the index page (results pasted below). This committed the style files in `docs/site-libs` ([see this commit](https://github.com/jdblischak/W_shredder_model/commit/60ef40b093ab5d8d6421a11dcf50861f64575bf6)). And now my fork displays the [website](https://jdblischak.github.io/W_shredder_model/index.html) properly.

Some questions to try and understand how this happened: Did you use `wflow_publish()`? Are you using a global `.gitignore` file that ignores CSS/JS files?

```
> wflow_publish("analysis/index.Rmd")
Current working directory: /home/jdb-work/repos/W_shredder_model
Building 1 file(s):
Building analysis/index.Rmd
Summary from wflow_publish

**Step 1: Commit analysis files**

No files to commit


**Step 2: Build HTML files**

Summary from wflow_build

Settings:
 clean_fig_files: TRUE

The following were built externally each in their own fresh R session: 

docs/index.html

Log files saved in /tmp/RtmpObrU9a/workflowr

**Step 3: Commit HTML files**

Summary from wflow_git_commit

The following was run: 

  $ git add docs/index.html docs/figure/index.Rmd docs/site_libs docs/.nojekyll 
  $ git commit -m "Build site." 

The following file(s) were included in commit 60ef40b:
docs/index.html
docs/site_libs/bootstrap-3.3.5/
docs/site_libs/font-awesome-5.1.0/
docs/site_libs/highlightjs-9.12.0/
docs/site_libs/jquery-1.11.3/
docs/site_libs/navigation-1.1/
```

**Comment by lukeholman** [comment]
That solved it, thanks! I don't how the `docs/site_libs` folder got lost from my Git. However I did not create this workflowr project in the normal way (using `wflow_start()`), so that probably explains it somehow. 

In case anyone gets the same issue, I solved it by adding `docs/site-libs` to the list of files tracked by Git, by typing `git add -f docs/site-libs/*` into the Terminal, where the -f forces git to add the files even though they were indeed listed in a .git.ignore file (not one that I created on purpose, but it was somewhere on my computer nonetheless!).
